samples/language/Operations.qs

```diff
@@ -0,0 +1,25 @@
+/// # Sample
+/// Operations   
+///
+/// # Description
+/// Operations are the basic building blocks of a Q# program. A Q#
+/// operation is a quantum subroutine. That is, it's a callable routine
+/// that contains quantum operations that modify the state of the
+/// qubit register.
+namespace MyQuantumApp {
+	
+    @EntryPoint()
+    operation MeasureOneQubit() : Result {
+        // Allocate a qubit, by default it is in zero state      
+        use q = Qubit();  
+        // We apply a Hadamard operation H to the state
+        // It now has a 50% chance of being measured 0 or 1  
+        H(q);      
+        // Now we measure the qubit in Z-basis.
+        let result = M(q);
+        // We reset the qubit before releasing it.
+        if result == One { X(q); }
```

Comment:
```suggestion
        // We reset the qubit before releasing it using the `Reset` operation.
        Reset(q);
```

Reply:
Resetting the qubit using the measurement result is logically correct but could not be executed on some hardware platforms (it implies branching based on measurement capabilities). It is more portable to use the `Reset` operations which works on all platforms. In general, when possible, our samples should aim to be as portable as possible.